Packs/FortiGate/Integrations/FortiGate/FortiGate.py

```diff
@@ -103,1209 +1503,2378 @@ def create_addr_string(list_of_addr_data_dicts):
+    """Ban IPs.
+
+    Args:
+        client (Client): Session to Fortigate to run API requests.
+        args (dict[str, Any]): Arguments passed down by the CLI to configure the request.
 
-    demisto.results({
-        'Type': entryTypes['note'],
-        'ContentsFormat': formats['json'],
-        'Contents': contents,
-        'ReadableContentsFormat': formats['markdown'],
-        'HumanReadable': tableToMarkdown('FortiGate policy with ID ' + policy_id + ' deleted successfully', contents),
-        'EntryContext': context
-    })
+    Returns:
+        CommandResults: Outputs of the command that represent an entry in warroom.
+    """
+    # response = client.ban_ip()
 
 
 @logger
-def delete_policy_request(policy_id):
-    uri_suffix = 'cmdb/firewall/policy/' + policy_id
-    response = http_request('DELETE', uri_suffix)
-    return response
+def unban_ip_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """Unban IPs.
+
+    Args:
+        client (Client): Session to Fortigate to run API requests.
+        args (dict[str, Any]): Arguments passed down by the CLI to configure the request.
+
+    Returns:
+        CommandResults: Outputs of the command that represent an entry in warroom.
+    """
+    # response = client.unban_ip()
+
+
+""" Deprecated Commands """
 
 
 @logger
-def get_address_groups_command():
+def get_addresses_command(client: Client, args: dict[str, Any]):
+    """DEPRECATED COMMAND"""
     contents = []
     context = {}
-    address_groups_context = []
-    address_group_name = demisto.args().get('groupName', '')
-    title = address_group_name if address_group_name else 'all'
-
-    address_groups = get_address_groups_request(address_group_name)
-    for address_group in address_groups:
-        members = address_group.get('member')
-        members_list = []
-        for member in members:
-            members_list.append(member.get('name'))
-        contents.append({
-            'Name': address_group.get('name'),
-            'Members': members_list,
-            'UUID': address_group.get('uuid')
-        })
-        address_groups_context.append({
-            'Name': address_group.get('name'),
-            'Member': {
-                'Name': members_list
-            },
-            'UUID': address_group.get('uuid')
-        })
+    addresses_context = []
+    address = args.get("address")
+    name = args.get("name", "")
 
-    context['Fortigate.AddressGroup(val.Name && val.Name === obj.Name)'] = address_groups_context
+    response = client.list_firewall_address_ipv4s(name, address)
 
-    demisto.results({
-        'Type': entryTypes['note'],
-        'ContentsFormat': formats['json'],
-        'Contents': contents,
-        'ReadableContentsFormat': formats['markdown'],
-        'HumanReadable': tableToMarkdown('FortiGate address groups ' + title, contents),
-        'EntryContext': context
-    })
+    if isinstance(response, list):
```

Comment:
`address == '*'` has no type validation which raises a warning in `mypy`, therefore I changed it to `if isinstance(response, list):`.

Reply:
Why are the conditions equivalent?

Reply:
When the API is given `"*"`, then it will return all the available response for each VDOM, which will always be a `list` response, even if there is only 1 VDOM (minimum).